packages/tessellate-fragment/src/bundle-service.js

```diff
@@ -22,15 +22,15 @@ class BundleProblem extends Problem {
   }
 }
 
-export async function fetchBundle(name: string): Promise<Bundle> {
-  const uri = url.parse(nconf.get('BUNDLE_SOURCE'))
+export async function fetchBundle(sources: Object): Promise<Bundle> {
```

Comment:
The `sources` argument is a well-defined type, so we could make a custom flow type of it instead of using just `Object`. Not necessary now, just an improvement for later.